.github/workflows/ci-pr.yaml

```diff
@@ -79,7 +79,7 @@ jobs:
         EOF
         echo Deploying application
         skaffold config set --global local-cluster false
-        skaffold run --default-repo=gcr.io/$PROJECT_ID/refs/pull/$PR_NUMBER --tag=$PR_NUMBER --namespace=$NAMESPACE
+        skaffold run --default-repo=gcr.io/$PROJECT_ID/refs/pull/$PR_NUMBER --tag=$PR_NUMBER --namespace=$NAMESPACE -p networkpolicy
```

Comment:
Praise:
Thanks for adding the `NetworkPolicy` `skaffold` patch to the pull-request CI!
I see that the NetworkPolicy objects have been applied on the cluster used by this pull-request's staging Online Boutique deployment.

Reply:
**Update:**
I have enabled NetworkPolicies on the GKE cluster being used for pull-requests:
```
gcloud container clusters update online-boutique-prs --update-addons=NetworkPolicy=ENABLED --zone us-central1-c
gcloud container clusters update online-boutique-prs --enable-network-policy --zone us-central1-c
```